backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/service/SAPGLJournalLineCreateRequest.java

```diff
@@ -16,10 +16,27 @@
 @Builder
 public class SAPGLJournalLineCreateRequest
 {
-	@NonNull SAPGLJournalId glJournalId;
 	@NonNull PostingSign postingSign;
 	@NonNull Account account;
 	@NonNull BigDecimal amount;
-	@Nullable SectionCodeId sectionCodeId;
+	@NonNull Dimension dimension;
 	@Nullable TaxId taxId;
+	@Nullable String description;
+	boolean determineTaxBaseSAP;
+
+	@NonNull
+	public static SAPGLJournalLineCreateRequest of(
+			@NonNull final SAPGLJournalLine line,
+			@NonNull final Boolean negateAmounts)
+	{
+		return SAPGLJournalLineCreateRequest.builder()
+				.postingSign(line.getPostingSign())
+				.account(line.getAccount())
+				.amount(line.getAmount().negateIf(negateAmounts).toBigDecimal())
```

Comment:
what about amountAcct?

Reply:
it is computed in `de.metas.acct.gljournal_sap.SAPGLJournal#addLine`